alarms.tf

```diff
@@ -134,11 +134,46 @@ resource "aws_cloudwatch_metric_alarm" "swap_usage_too_high" {
   period              = var.period
   statistic           = "Average"
   threshold           = local.thresholds["SwapUsageThreshold"]
-  alarm_description   = "Average database swap usage over last ${var.period/60} minutes too high, performance may suffer"
+  alarm_description   = "Average database swap usage over last ${var.period / 60} minutes too high, performance may suffer"
   alarm_actions       = [var.aws_sns_topic_arn]
   ok_actions          = [var.aws_sns_topic_arn]
 
   dimensions = {
     DBInstanceIdentifier = var.db_instance_ids[count.index]
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "rapid-free-space-decrease" {
+  count                     = length(var.db_master_ids)
+  alarm_name                = "${var.name_prefix}${var.db_master_ids[count.index]}-rapid-free-space-decrease"
+  comparison_operator       = "LessThanLowerThreshold"
+  evaluation_periods        = "2"
+  threshold_metric_id       = "e1"
+  alarm_description         = "RDS Free storage space"
+  insufficient_data_actions = [var.aws_sns_topic_arn]
+  ok_actions                = [var.aws_sns_topic_arn]
+  alarm_actions             = [var.aws_sns_topic_arn]
+
+  metric_query {
+    id          = "e1"
+    expression  = "ANOMALY_DETECTION_BAND(m1)"
+    label       = "FreeStorageSpace (Expected)"
+    return_data = "true"
```

Comment:
not sure why both metric queries in the example from the official docs have `return_data` set here, because the same docs suggest only one should have them :shrug: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#example-with-an-expression